config/overrideChrome.js

```diff
@@ -1,5 +1,4 @@
 export default () => ({
   updateDocumentTitle: () => undefined,
-  isBeta: () => true,
```

Comment:
isBeta is not going away according to the platform. Should we leave as is this for now

Reply:
It was not used in the code so it did not need to be overridden, but I'll keep it for future use.

Reply:
Hm, actually, [`isBeta` is deprecated](https://github.com/RedHatInsights/insights-chrome/blob/8205b3043192f297f05e0c94444bd75f1ca0f722/src/utils/VisibilitySingleton.ts#L90) and should not be used anymore, so it does make sense to remove it